.github/workflows/translations.yaml

```diff
@@ -0,0 +1,63 @@
+name: Automated syncronization and update of translations
+
+on:
+  shedule:
+    branches: [ master ]
+    - cron:  '0 0 */3 * *'
+
+jobs:
+  translations_sync_and_update:
+    name: Sync translations from Transifex and update source .ts files with new unstranslated strings
+
+    runs-on: ubuntu-latest
+
+    steps:
+    - name: checkout repository
+      uses: actions/checkout@v2.3
```

Comment:
The latest version is v2.3.4.

Reply:
Do we really need to pin specific version? Isn't the 2.3.x branch stable enough?

Reply:
>Do we really need to pin specific version? 

I could be wrong, v2.3 is equal to v2.3.0, it is a specific version, isn't? If so, then why not use the latest one?
Or you can omit `@v2.3` part to always use the latest release.

Reply:
Seems I misunderstood it: https://docs.github.com/en/free-pro-team@latest/actions/reference/workflow-syntax-for-github-actions#jobsjob_idstepsuses

Then how about we pin to the major version `v2`? The doc says:
>Using the specific major action version allows you to receive critical fixes and security patches while still maintaining compatibility. It also assures that your workflow should still work.